python/lsst/ts/m2/csc.py

```diff
@@ -276,7 +276,12 @@ async def _publish_message_by_sal(
 
         if hasattr(self, sal_topic_name):
             message_payload.pop("id")
-            await getattr(self, sal_topic_name).set_write(**message_payload)
+
+            try:
+                await getattr(self, sal_topic_name).set_write(**message_payload)
+            except Exception as error:
+                self.log.debug(f"Error in publishing data: {error!r}.")
```

Comment:
You may want to log the full exception no?

```
except Exception:
    self.log.exception(f"Error publishing {sal_topic_name}.")
```

Reply:
This would be too much for the telemetry in `verbose`...
The information in `{error!r}` is good enough for debugging.